action.yml

```
@@ -7,14 +7,17 @@ inputs:
     description: 'Input string that gets passed along to the docker build args'
     required: true
   tag_list:
-    description: 'List of tags that gets passed along to the docker build args'
+    description: 'List of comma-separated tags that gets passed along to the docker build args'
     required: true
     default: '' # set to an empty string
-  docker_username:
-    description: 'Docker username obtained from secrets'
+  registry_url:
+    description: 'Url of the registry action is loging in to'
     required: true
-  docker_password:
-    description: 'Docker password obtained from secrets'
+  registry_user:
+    description: 'Username of the registry obtained from secrets'
```

```suggestion
    description: 'Username of the registry'
```

Technically this doesn't have to be a secret